sbws/lib/resultdump.py

```diff
@@ -82,7 +82,44 @@ def trim_results(fresh_days, result_dict):
     return out_results
 
 
-def load_recent_results_in_datadir(fresh_days, datadir, success_only=False):
+def trim_results_ip_changed(result_dict, ipv4=True, ipv6=False):
```

Comment:
Please rename ipv4/ipv6 to be more descriptive. For example, `trim_changed_ipv4` or `on_changed_ipv6`

Reply:
Consider making both default to False (but leaving the ipv4 option True in config.default.ini